### DISCUSSION TITLE
VU OLED configuration options/files

### DISCUSSION THREAD

**Discussion by noriadzik** [rern/rAudio#1492] [General]
I would like to ask, is anyone know, where can I change OLED options? Specially, I would like to modify:
- OLED screen brightness
- bars quantity
- bars "response time" to music (to rise/fall faster)
- have possibility to turn off / turn on bars (to be able to put "black screen", to have "night mode")

**Comment by kan-ibal** [comment]
Have a look at [mpd_oled](https://github.com/rern/rAudio/discussions/162)

**Reply by noriadzik** [reply]
Thanks for suggestion, I know mpd_oled. And well - I failed to install it on raspberry with rAudio OS, so I was looking around for other possibilities to match my needs ;-)
